### PR TITLE
bugfix/button onclick typo

### PR DESCRIPTION
**Pull request by 01231**
Fixes bug, where no forms can be submitted in Edge.

# Done is when (DoD):
- [ ] I have added UI/Unit tests for my changes
- [ ] Design has been reviewed with (here name of reviewer)
- [x] Old tests and eslint rule are not broken
- [ ] My changes are well documented in the readme and showcased in the stories 

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas (focus on **WHY**)
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by raphaellueckl** [comment]
hombre da isch ja alles kabutt, was sölli denn do reviewe?

File: src/components/10-atoms/button/index.js [hunk 1]
```diff
@@ -127,7 +127,7 @@ class AXAButton extends LitElement {
   disconnectedCallback() {
     super.disconnectedCallback();
 
-    if (this.isTypeSubmitOrReset) this.onclick = null;
+    if (this.isTypeSubmitOrReset) this.onClick = null;
```
Comment (danyball) [inline src/components/10-atoms/button/index.js]: a `if` without `{}` is bad practise. Pls add them.

Comment (01231) [reply src/components/10-atoms/button/index.js]: I will🥶

**Review by danyball — COMMENTED** [review]
Generally: i dont think that this is our problem at AEM forms. It is definitely a typo, but Line 67 is running anyway, isnt it?? Its worth trying.

**Comment by 01231** [comment]
So I found out that `onclick` is the event. When the `axa-button` gets clicked it clicks the fake button instead. `onClick` refers to the function-valued attribute. They have nothing in common. It wouldn't make sense that this is a typo, because then the fake-button stuff would only apply to react and other frameworks.

File: src/components/10-atoms/button/index.js [hunk 2]
```diff
@@ -58,13 +58,23 @@ class AXAButton extends LitElement {
 
   firstUpdated() {
     const { style } = this;
+    const isNativeShadowDOM = /\{\s*\[native code\]\s*\}/.test(
```
Comment (markus-walther) [inline src/components/10-atoms/button/index.js]: Too complicated: how about ``${ShadowRoot}`.indexOf('native code') > -1`?

**Review by markus-walther — CHANGES_REQUESTED** [review]

**Review by markus-walther — APPROVED** [review]
LGTM

**Pull request merged**